src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: #000;
+}
+
+.container,
+.box_orange,
+.box_content {
+  border: 1px solid #e95d49;
+  padding: 10px;
+}
+
```

Comment:
Rearrange these styles between corresponding selectors.
All styles for the class should be under one selector:
```
.container {
  styles go here
}

.box_orange {
  styles go here
}
```
etc.

Reply:
These styles are grouped because they are repeated for these classes. Otherwise, I will have to repeat this code three times -for each selector. I suppose, such behavior violates the DRY rule.

Reply:
Usually, there should be only one instance of certain selector per stylesheet.

And yes, you are correct. In this case the best option is to create one more common class, like `block` and apply these styles to it. As you may know, you can add more than one class to HTML-element.